smarts/env/wrappers/episode_logger.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2022. Huawei Technologies Co., Ltd. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+from typing import Any, Dict, Tuple
+
+import gym
+
+from smarts.core.utils.episodes import EpisodeLog, EpisodeLogs
+
+Action = Any
+Operation = Any
+
+
+class EpisodeLogger(gym.Wrapper):
+    """Wraps a gym environment with simple episode logging capabilities."""
+
+    def __init__(self, env: gym.Env, col_width: int = 18):
+        super(EpisodeLogger, self).__init__(env)
+        self._current_episode = None
+        self._done = False
```

Comment:
Perhaps `self._closed` instead of `self._done`.
```suggestion
        self._closed = False
```